### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by fishstik** [dcchambers/vb4c#23]
Has this fork been abandoned? Still no web store publication or source code updates for over a year now.

Maybe someone who actually intends to work on it can try again?

**Comment by dcchambers** [comment]
Hi @fishstik,

To be blunt: unfortunately...yeah pretty much. At least "indefinite hiatus."

Some background:

I forked the cVim project at the beginning of March 2020 when I noticed the broken `hints` feature was preventing the extension from doing what I used it for most. I'm not a javascript developer, but some people had already submitted patches to the original project to fix the issue. The creator/maintainer of that project didn't seem to be around any more, so I forked it and merged in a couple of patches to get it functioning again. I had some grand plans about fixing all of the outdated bugs in the code and helping to maintain this project but then, well, March 2020...a certain little virus swept us all onto our collective asses and the world was turned upside down.

After merging in a couple of code fixes, I submitted the new project twice to the google chrome store but was rejected without any real indication as to what the reason was. The code, other than a few lines from those fixes, was identical to the cVim project which was allowed on the store. I was getting ready to dig into the source code to try and figure out exactly what permissions it needed and to be able to prove them to the review teem at Google so I could get the app listed when COVID hit and my time just kind of vanished. At the same time, I did get the app fixed and it was working for me how I wanted it to, so as long as you were OK doing a manual install, it was "good enough" where it wasn't so much of a priority for me.

Combine that with the rapid changes at work going on in the Spring of 2020 because of COVID and I just completely ran out of time. My technical chops are in backend and infrastucture code, so javascript is like the wild west for me. The existing cVim code is very poorly documented and is quite old, especially by javascript standards. I always hoped to return to this and fix it in earnest, I just haven't had the time.

**TL;DR** - If someone wants to fork this or fork the original cVim project and take another stab at maintaining a fork, that's probably the best way forward. I do hope to be able to find the time to work on this again or at least support a properly maintained fork. As for what is best to do with this repo *right now*? I'm not sure. I should probably at least update the README to update the status...and realistically I should probably archive it until/unless I'm able to maintain it or add some maintainers.

**Comment by fishstik** [comment]
Thanks for the update, and understandable given Mr. Bones' Wild Ride 2020 edition.

**Comment by gonsalu** [comment]
Seems like even though there's no commits in cVim repo, the webstore got updated a couple of days ago and it's working again...

**Comment by dcchambers** [comment]
Since the original https://github.com/1995eaton/chromium-vim project has been updated to fix the "hints" feature, and I have not had the time to be an active maintainer of this project, I will be placing this repo into archived status.
